docs/getting_started/index.rst

```diff
@@ -1,11 +1,46 @@
 Getting started
 ===============
 
+Wagtail is built on the `Django web framework <https://www.djangoproject.com/>`_, so this document assumes you've already got the essentials installed. But if not, those essentials are:
+
+ * `Python <https://wiki.python.org/moin/BeginnersGuide/Download>`_
+ * `pip <https://pip.pypa.io/en/latest/installing.html>`_ (Note that pip is included by default with Python 2.7.9 and later and Python 3.4 and later)
+
+We'd also recommend:
+
+ * `Virtualenv <https://virtualenv.pypa.io/en/latest/installation.html>`_
+
+
+With the above installed and enabled, the quickest way to install Wagtail is::
+
+    pip install wagtail
```

Comment:
sudo isn't the answer to all problems though. Why not explain that if you're using a unix box and want to install it system-wide, use sudo?